src/app/tests/TestClusterObjectUtils.cpp

```diff
@@ -0,0 +1,782 @@
+/*
```

Comment:
Maybe the file/class name could be changed to TestClusterObjectCommands, as it provides a test suite for cluster object commands?

Reply:
How about Serialization instead of Utils in the name?

Utils is very vague - it is unclear what 'utilities for cluster objects' would contain.